plugins/LilyPad/InputManager.cpp

```diff
@@ -433,7 +433,6 @@ Device *InputDeviceManager::GetActiveDevice(InitInfo *info, unsigned int *uid, i
 							  (devices[i]->oldVirtualControlState[j] > 31*FULLY_DOWN/32 && devices[i]->virtualControlState[j] < 7*FULLY_DOWN/8))) {
 									continue;
 						}
-						devices[i]->virtualControls[j].uid = devices[i]->virtualControls[j].uid;
```

Comment:
Nope, no ideas here.

I assume it's code left over from someone rearranging the code, considering the last two parameters of GetActiveDevice aren't used for anything at all (it's only called in one place, and that doesn't use the variables).